lib/cpu_color_spinor_field.cpp

```diff
@@ -330,4 +330,16 @@ namespace quda {
     host_free(sendbuf);
   }
 
+  void cpuColorSpinorField::copy_to_buffer(void *buffer) const
+  {
+    std::memcpy(buffer, v, bytes);
+    if (precision < QUDA_SINGLE_PRECISION) { std::memcpy(static_cast<char *>(buffer) + bytes, norm, norm_bytes); }
```

Comment:
Should we have it barf here if `precision < QUDA_SINGLE_PRECISION` is true, since CPU fields don't support fixed precision formats yet? Or is this just a forward-thinking call?

Reply:
This is a forward proving call, and I guess there is no harm to keep it here?